manager/manager.go

```diff
@@ -184,6 +184,10 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 		inHostNamespace = true
 	}
 
+	if ignoreMetricsSet == nil {
+		ignoreMetricsSet = ignoreMetrics.MetricSet
```

Comment:
I'd prefer that this was explicitly passed in from the main module rather than implicitly set here.